common/src/main/java/org/apache/comet/NativeBase.java

```diff
@@ -46,15 +46,24 @@ public abstract class NativeBase {
 
   private static final String libraryToLoad = System.mapLibraryName(NATIVE_LIB_NAME);
   private static boolean loaded = false;
+  private static Throwable loadErr = null;
```

Comment:
Does this need to be `volatile`?

Reply:
Good point. I have made it volatile now.